tfhe/src/core_crypto/algorithms/lwe_multi_bit_programmable_bootstrapping.rs

```diff
@@ -1343,8 +1115,6 @@ pub fn std_multi_bit_blind_rotate_assign<Scalar, InputCont, OutputCont, KeyCont>
 
     assert!(work_queue.len() == lwe_mask.lwe_dimension().0 / grouping_factor.0);
 
-    let work_queue = Mutex::new(work_queue);
```

Comment:
I cannot comment directly on it but I see a ggsw_vec comment line 1103 of the new code that could likely be removed